src/timers.cc

```diff
@@ -127,7 +138,8 @@ double ProcessCPUUsage() {
 double ThreadCPUUsage() {
 // FIXME We want to use clock_gettime, but its not available in MacOS 10.11. See
 // https://github.com/google/benchmark/pull/292
-#if defined(CLOCK_THREAD_CPUTIME_ID) && !defined(BENCHMARK_OS_MACOSX)
+#if defined(CLOCK_THREAD_CPUTIME_ID) && !defined(BENCHMARK_OS_MACOSX) && \
```

Comment:
This case feels like the `#else` case., but it definitely shouldn't be the primary `#if`. Please restructure these ifdefs so that the most specialized implementations are at the top, and the generic one is the `#else`.